packages/api/endpoints/collections.js

```diff
@@ -130,28 +173,39 @@ async function post(req, res) {
  * @param {Object} res - express response object
  * @returns {Promise<Object>} the promise of express response object
  */
-async function put({ params: { name, version }, body }, res) {
-  if (name !== body.name || version !== body.version) {
+async function put(req, res) {
+  const { name, version } = req.params;
+  const collection = req.body;
+
+  if (name !== collection.name || version !== collection.version) {
     return res.boom.badRequest('Expected collection name and version to be'
-      + ` '${name}' and '${version}', respectively, but found '${body.name}'`
-      + ` and '${body.version}' in payload`);
+      + ` '${name}' and '${version}', respectively, but found '${collection.name}'`
+      + ` and '${collection.version}' in payload`);
   }
 
-  const collectionModel = new models.Collection();
+  const collectionsModel = new models.Collection();
 
-  if (!(await collectionModel.exists(name, version))) {
+  if (!(await collectionsModel.exists(name, version))) {
     return res.boom.notFound(
       `Collection '${name}' version '${version}' not found`
     );
   }
 
-  const record = await collectionModel.create(body);
+  const dynamoRecord = await collectionsModel.create(collection);
+
+  const dbRecord = dynamoRecordToDbRecord(dynamoRecord);
+
+  const dbClient = await getKnexClient();
```

Comment:
How is this working?   We don't have a mapping between `databaseCredentialSecretArn = var.rds_user_access_secret_arn`, and getKnexClient has no defaults to work against in the current TF environment variables.

Reply:
Or.... there is.     Resolving.